exporters/stdout/stdoutmetric/config.go

```diff
@@ -26,7 +26,6 @@ var (
 	defaultPrettyPrint         = false
 	defaultTimestamps          = true
 	defaultLabelEncoder        = attribute.DefaultEncoder()
-	defaultDisableTraceExport  = false
 	defaultDisableMetricExport = false
```

Comment:
These made sense when this was a combined exporter.  Should we just remove these options?